modules/skybrowser/include/utility.h

```diff
@@ -0,0 +1,81 @@
+ * OpenSpace                                                                             *
+ *                                                                                       *
+ * Copyright (c) 2014-2022                                                               *
+ *                                                                                       *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this  *
+ * software and associated documentation files (the "Software"), to deal in the Software *
+ * without restriction, including without limitation the rights to use, copy, modify,    *
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to    *
+ * permit persons to whom the Software is furnished to do so, subject to the following   *
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#ifndef __OPENSPACE_MODULE_SKYBROWSER___UTILITY___H__
+#define __OPENSPACE_MODULE_SKYBROWSER___UTILITY___H__
+
+#include <openspace/documentation/documentation.h>
+
+namespace openspace::skybrowser {
+// Constants
+constexpr const double ScreenSpaceZ = -2.1;
+constexpr const glm::dvec3 NorthPole = { 0.0 , 0.0 , 1.0 };
+constexpr const double CelestialSphereRadius = std::numeric_limits<float>::max();  
+
+// Conversion matrix - J2000 equatorial <-> galactic
+// https://arxiv.org/abs/1010.3773v1
+const glm::dmat3 conversionMatrix = glm::dmat3({
+            -0.054875539390,  0.494109453633, -0.867666135681, // col 0
+            -0.873437104725, -0.444829594298, -0.198076389622, // col 1
+            -0.483834991775,  0.746982248696,  0.455983794523 // col 2
+    });
+
+// Galactic coordinates are projected onto the celestial sphere
+// Equatorial coordinates are unit length
+// Conversion spherical <-> Cartesian
+glm::dvec2 cartesianToSpherical(const glm::dvec3& coords);
```

Comment:
Seems like these utility functions could be useful in other cases :) Could live in https://github.com/OpenSpace/OpenSpace/blob/master/include/openspace/util/coordinateconversion.h ?

Also, I feel like all these functions deserve a proper doxygen comment/description (see examples in file linked above)

Reply:
Should I move the functions there? 🤔